src/faucet.js

```diff
@@ -1,45 +1,49 @@
 const Web3 = require("web3");
 require('dotenv').config();
 const axios = require("axios");
-const jsonInterface = require('../src/abi.json');
-const { SSV_INFURA_HTTPS_ENDPOINT, SSV_EXPLORER_URL, SINGER_PRIVATE_KEY, SIGNER_OWNER_ADDRESS, SSV_CONTRACT_ADDRESS } = process.env;
-const web3 = new Web3(new Web3.providers.HttpProvider(SSV_INFURA_HTTPS_ENDPOINT));
+const { getContractToken, NETWORK_ABI } = require("./envHelper");
+const {  SSV_EXPLORER_URL, SINGER_PRIVATE_KEY, SIGNER_OWNER_ADDRESS } = process.env;
```

Comment:
Should signer private key and owner address be per network?
As soon as I know Andrew told us it will be different accounts

Reply:
no, I spoke with Andrew it should be the same wallet for both networks